buildSrc/subprojects/documentation/src/main/groovy/gradlebuild/docs/GradleUserManualPlugin.java

```diff
@@ -165,7 +165,7 @@ private void generateUserManual(Project project, TaskContainer tasks, ProjectLay
             // TODO: This breaks if the version is changed later.
             attributes.put("gradleVersion", project.getVersion().toString());
             attributes.put("snippetsPath", "snippets");
-            attributes.put("samplesPath", "samples");
+            attributes.put("samplesPath", extension.getUserManual().getStagingRoot().dir("raw/samples").get().getAsFile());
```

Comment:
❌ I'm not sure this is necessary. `samplesPath` should already point to the same directory: https://github.com/gradle/gradle/blob/ad40fb2cfa852afb28695ffd03f248ec41c4ccbd/subprojects/docs/src/docs/userguide/dep-man/05-multirepo-environment/composite_builds.adoc#L52

Reply:
See comment on commit: https://github.com/gradle/gradle/pull/14291/commits/37344919085e511e7784050a84f7c0f7b71ffe2f

Fix path to samples working dir so that also the 'samplesMultiPage' task finds it into the
'userguide/raw' working dir.

Otherwise 'samplesPath' in the context of 'samplesMultiPage' point at 'build/samples' and not 'build/userguide/raw'

Fixes the problem that blocks #14131

Reply:
This feels a bit hacky and imho should be fixed in the samples plugin. I'm fine with the change though. Could you please add a few line of the comments explaining this isssue?

Reply:
This feels a bit hacky and IMHO should be fixed in the samples plugin. I'm fine with the change though. Could you please add a few lines of the comments explaining this issue?

Reply:
I added a comment

Reply:
👍